src/components/composer/composer/composer.ts

```diff
@@ -268,37 +262,32 @@ export class Composer extends Component<ComposerProps, SpreadsheetChildEnv> {
         this.autoComplete(autoCompleteValue);
         return;
       }
-    } else {
-      // when completing with tab, if there is no value to complete, the active cell will be moved to the right.
-      // we can't let the model think that it is for a ref selection.
-      // todo: check if this can be removed someday
-      this.env.model.dispatch("STOP_COMPOSER_RANGE_SELECTION");
```

Comment:
Probably still needed, not 100% sure it's because of this but we can lose the topbar composer : 
- type = 1 + in the topbar composer
- press tab
- the selection move, and the topbar composer is stuck

Reply:
can't reproduce :/